### PR TITLE
Added content mode management for display image on iOS

### LINKED ISSUE #96: Add an option to center or fit background image for iOS App Launcher
**Is your feature request related to a problem? Please describe:**

At the moment on iOS the background image passed in config is centered:

`imageView.contentMode = .center
`

**Describe the solution you'd like:**

In our case we need to fit the image provided to the view controller, otherwise it is too zoomed in:

`imageView.contentMode = .scaleAspectFit`

It would be very useful to have an option that can be passed in config, something like 

`    PrivacyScreen: {
      enable: true,
      imageName: "Splash",
      contentMode: "[fit/center]"
    }`

**Describe alternatives you've considered:**

The only alternative we have right now is to fork out and customize the plugin, losing the option of keeping up with the updates.

### PR DESCRIPTION
**Pull request by nomadminded**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] The changes have been tested successfully.

Close #96

### CODE REVIEW

File: src/definitions.ts [hunk 1]
```diff
@@ -26,6 +26,17 @@ declare module '@capacitor/cli' {
...
        */
       imageName?: string;
+      /**
+       * Configure the content mode of displayed image. 
+       * Accepts values "center", "scaleToFill", "scaleAspectFit", "scaleAspectFill".
+       * Defaults to UIView.ContentMode.center in case of a different value or no value provided.
+       *
+       * Only available for iOS.
+       *
+       * @default ""
```
Comment (robingenz) [inline src/definitions.ts:36]: Shouldn't that be "center"?

File: src/definitions.ts [hunk 2]
```diff
@@ -26,6 +26,17 @@ declare module '@capacitor/cli' {
...
+       * @example "Splashscreen"
```
Comment (robingenz) [inline src/definitions.ts:37]: This example value does not exist.

File: src/definitions.ts (hunk 1 above)
Comment (nomadminded) [reply src/definitions.ts:36]: Good catch, thank you! Will update asap

File: src/definitions.ts (hunk 2 above)
Comment (nomadminded) [reply src/definitions.ts:37]: Good catch, thank you! Will update asap

File: src/definitions.ts (hunk 1 above)
Comment (nomadminded) [reply src/definitions.ts:36]: I'd leave it blank as the value is not required, if not provided it automatically defaults to center. Just to leave it backward compatible as well.

**Pull request merged**